README.md

```diff
@@ -7,6 +7,19 @@
 [![Devices](https://img.shields.io/badge/devices-iPhone%20%7C%20iPad-blue.svg)]()
 [![Swift version](https://img.shields.io/badge/swift-5.0-orange.svg)]()
 
+**Deprecation Notice**
+ ----------------------
+
+ Development of the Gini Capture SDK for iOS will be continued in a 
+ [new repository](https://github.com/gini/capture-sdk-ios).
+
+ No new versions will be released from this repository and we kindly ask you to update to the 
+ [version](https://github.com/gini/capture-sdk-ios) released from the new repository.
```

Comment:
Version link does not point to any specific release. My intent with this was to point to the first public version released from the monorepo so they know exactly which version to update to.

The first public version of the Capture SDK swift package would be 1.1.0 ([1.0.7](https://github.com/gini/gini-capture-sdk-ios/releases) is the latest we released for CocoaPods).

Reply:
Ok, I'll add this. 😊